client/src/Pages/LoginScreen.jsx

```diff
@@ -1,18 +1,21 @@
 import React from 'react';
-import {Input, Button} from 'antd';
-import { Link } from 'react-router-dom';
+import {Row, Col} from 'antd';
+import LoginForm from './LoginForm';
 
 export default class LoginScreen extends React.Component{
```

Comment:
Just a general warning: I know very little about class components in React; I've only used functional components. So for handling any of the weird async WebSocket stuff in a class component, I won't be of much help.

However, you're obviously the front-end person. So if you know way more about class components than functional ones, just stick with them.